src/modules/components/proposal/proposalVoting/proposalVotingContainer/proposalVotingContainer.tsx

```diff
@@ -1,34 +1,22 @@
 import classNames from 'classnames';
 import React, { Children, type ComponentProps } from 'react';
-import { Accordion, Card, Heading } from '../../../../../core';
+import { Accordion, Card } from '../../../../../core';
 
 export interface IProposalVotingContainerProps extends ComponentProps<'div'> {
-    /**
-     * Title displayed on top.
-     */
-    title: string;
-    /**
-     * Description of the proposal voting.
-     */
-    description: string;
     /**
      * Active stage that will be expanded by default for multi-stage proposals.
      */
     activeStage?: string;
 }
 
 export const ProposalVotingContainer: React.FC<IProposalVotingContainerProps> = (props) => {
-    const { title, description, className, children, activeStage, ...otherProps } = props;
+    const { className, children, activeStage, ...otherProps } = props;
 
     const processedChildren = Children.toArray(children);
     const isMultiStage = processedChildren.length > 1;
 
     return (
-        <Card className={classNames('flex flex-col overflow-hidden', className)} {...otherProps}>
-            <div className={classNames('flex flex-col gap-2 p-4 md:gap-3 md:p-6', { 'pb-2 md:pb-3': !isMultiStage })}>
-                <Heading size="h2">{title}</Heading>
-                <p className="text-base font-normal leading-normal text-neutral-500">{description}</p>
-            </div>
+        <Card className={classNames('flex w-full flex-col overflow-hidden', className)} {...otherProps}>
```

Comment:
The full component alignment with the new design can be done later, but without the title and description the component looks off. We can probably update the paddings and gap to have a better temporary version of the component
<img width="619" alt="Screenshot 2024-11-04 at 15 50 42" src="https://github.com/user-attachments/assets/28ad20c4-166a-449f-a57e-8be2d766b4e6">